src/widgets/ActivityDetails/ui/AssessmentPassingScreen.tsx

```diff
@@ -145,11 +149,12 @@ export const AssessmentPassingScreen = (props: Props) => {
     }
 
     if (!hasNextStep) {
-      return onSubmit()
+      return setIsModalOpen(true)
+      // return onSubmit()
```

Comment:
Should we remove this? Seems unnecessary to leave it around

```suggestion
```

Reply:
Oh, I forgot to remove it. Thank you!